### DISCUSSION TITLE
Rare bug of friend No. suddenly jumps to 1000+

### DISCUSSION THREAD

**Discussion by blufish1234** [vrcx-team/VRCX#810] [Q&A]
![image](https://github.com/vrcx-team/VRCX/assets/97332474/329cd909-a391-448d-a7bb-d73b56a4d79e)
I don't know but I opened VRCX window today (The app has been running in background for a few days) and suddenly discovered I have more than a thousand friends. I certainly do know that I don't have that many, so I checked and found out this. I don't know if it can be reproduced and never encountered this before. It returned to normal when I restart the application. Should I report this as a bug?

**Comment by Natsumi-sama** [comment]
were they new friends added from that session?

**Reply by blufish1234** [reply]
I haven't added any friends during that session
